app/helpers/welcome_helper.rb

```diff
@@ -0,0 +1,63 @@
+module WelcomeHelper
+  def greeting
+    if first_name
+      "Hey #{first_name}"
+    else
+      "Hello"
```

Comment:
I can fix this by putting it all on one line, but it will break the design somewhat.

Reply:
Does it all have to be classed as h1?

This might be creating too much work but this doesn't really feel like h1 content

eg 'Welcome to Teaching' could be the h1

followed by the rest of the content as just (styled) para text
eg Hey xxxx - thanks for signing up

However, I suspect this might be painful in terms of styling - let's have a quick chat before you do anything

Reply:
What you suggest sounds fine and should be straightforward to style, I'll implement it 👍🏽

Reply:
Done in 5050549c6